### ISSUE TITLE
Can't import react library from express app

### ISSUE THREAD

**Issue by goldylucks** [nrwl/nx#12578]
## Current Behavior

When importing a component from a react library, from an express app, I get this:

```
'@nx/admin-ui-components' is imported by apps/admin-ui/src/TestInner.ts, but could not be resolved – treating it as an external dependency
```

This is a freshly generated library by nx generator.

I need react component on an express app since I'm using adminjs, which runs on express but renders react components

## Expected Behavior

Should just work

## Steps to Reproduce

1. generate express app
2. generate ui react library
3. import library into app
4. run serve (build works just fine)

I also tried making the library buildable, but it didn't help

### Environment

```
$ npx nx report
npx nx report

 >  NX   Report complete - copy this into the issue template

   Node : 17.6.0
   OS   : darwin arm64
   yarn : 1.22.18

   nx : 14.4.3
   @nrwl/angular : Not Found
   @nrwl/cypress : 14.4.3
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.4.3
   @nrwl/eslint-plugin-nx : 14.4.3
   @nrwl/express : 14.4.3
   @nrwl/jest : 14.4.3
   @nrwl/js : 14.4.3
   @nrwl/linter : 14.4.3
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : 14.4.3
   @nrwl/nx-cloud : 14.2.0
   @nrwl/nx-plugin : Not Found
   @nrwl/react : 14.4.3
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.4.3
   @nrwl/web : 14.4.3
   @nrwl/workspace : 14.4.3
   typescript : 4.7.4
   ---------------------------------------
   Community plugins:
```

**Comment by FrozenPandaz** [comment]
Could you please provide a repo where we can see this issue? It would help us resolve the issue quicker.

**Comment by goldylucks** [comment]
closed in favor of #12634
